backend/lcfs/web/api/notional_transfer/validation.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import Depends, HTTPException, Request
```

Comment:
Nice to have this validation class! It keeps validation logic organized and centralized. 👍